diff --git a/config.h b/config.h
index a075887..69d0ae0 100644
--- a/config.h
+++ b/config.h
@@ -5,16 +5,18 @@
 #define TERMCLASS "St"
 
 /* appearance */
-static unsigned int borderpx  = 1;        /* border pixel of windows */
-static unsigned int snap      = 30;       /* snap pixel */
-static unsigned int gappih    = 5;       /* horiz inner gap between windows */
-static unsigned int gappiv    = 5;       /* vert inner gap between windows */
-static unsigned int gappoh    = 5;       /* horiz outer gap between windows and screen edge */
-static unsigned int gappov    = 5;       /* vert outer gap between windows and screen edge */
-static int swallowfloating    = 0;        /* 1 means swallow floating windows by default */
-static int smartgaps          = 0;        /* 1 means no outer gap when there is only one window */
-static int showbar            = 1;        /* 0 means no bar */
-static int topbar             = 1;        /* 0 means bottom bar */
+static unsigned int borderpx    = 1;        /* border pixel of windows */
+static unsigned int snap        = 30;       /* snap pixel */
+static unsigned int gappih      = 5;        /* horiz inner gap between windows */
+static unsigned int gappiv      = 5;        /* vert inner gap between windows */
+static unsigned int gappoh      = 5;        /* horiz outer gap between windows and screen edge */
+static unsigned int gappov      = 5;        /* vert outer gap between windows and screen edge */
+static unsigned int horizpadbar = 4;        /* horizontal padding for statusbar */
+static unsigned int vertpadbar  = 2;        /* vertical padding for statusbar */
+static int swallowfloating      = 0;        /* 1 means swallow floating windows by default */
+static int smartgaps            = 0;        /* 1 means no outer gap when there is only one window */
+static int showbar              = 1;        /* 0 means no bar */
+static int topbar               = 1;        /* 0 means bottom bar */
 /* install fonst
  * yay -Sy libxft-bgra
  * yay -Sy ttf-joypixels
diff --git a/dwm.c b/dwm.c
index bddab5f..5688363 100644
--- a/dwm.c
+++ b/dwm.c
@@ -886,8 +886,9 @@ drawbar(Monitor *m)
 	/* draw status first so it can be overdrawn by tags later */
 	if (m == selmon) { /* status is only drawn on selected monitor */
 		drw_setscheme(drw, scheme[SchemeNorm]);
-		tw = TEXTW(stext) - lrpad + 2; /* 2px right padding */
+		tw = TEXTW(stext) - lrpad;
 		drw_text(drw, m->ww - tw, 0, tw, bh, 0, stext, 0);
+
 	}
 
 	for (c = m->clients; c; c = c->next) {
@@ -1795,8 +1796,8 @@ setup(void)
 	drw = drw_create(dpy, screen, root, sw, sh);
 	if (!drw_fontset_create(drw, fonts, LENGTH(fonts)))
 		die("no fonts could be loaded.");
-	lrpad = drw->fonts->h;
-	bh = drw->fonts->h + 2;
+	lrpad = drw->fonts->h + horizpadbar;
+	bh = drw->fonts->h + vertpadbar;
 	updategeom();
 	/* init atoms */
 	utf8string = XInternAtom(dpy, "UTF8_STRING", False);
diff --git a/patch/dwm-titlecolor-20190206-6.2.diff b/patch/dwm-titlecolor-20190206-6.2.diff
deleted file mode 100644
index fecf8f3..0000000
--- a/patch/dwm-titlecolor-20190206-6.2.diff
+++ /dev/null
@@ -1,47 +0,0 @@
-From b15016b717c88eb378d87703dcf2169b6059047c Mon Sep 17 00:00:00 2001
-From: knary <theknary@gmail.com>
-Date: Wed, 6 Feb 2019 21:44:19 -0500
-Subject: [PATCH] Adds title bar color scheme, seperating it from SchemeSel.
-
----
- config.def.h | 1 +
- dwm.c        | 4 ++--
- 2 files changed, 3 insertions(+), 2 deletions(-)
-
-diff --git a/config.def.h b/config.def.h
-index 1c0b587..48e7ace 100644
---- a/config.def.h
-+++ b/config.def.h
-@@ -16,6 +16,7 @@ static const char *colors[][3]      = {
- 	/*               fg         bg         border   */
- 	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
- 	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
-+	[SchemeTitle]  = { col_gray4, col_cyan,  col_cyan  },
- };
- 
- /* tagging */
-diff --git a/dwm.c b/dwm.c
-index 4465af1..216e1f3 100644
---- a/dwm.c
-+++ b/dwm.c
-@@ -59,7 +59,7 @@
- 
- /* enums */
- enum { CurNormal, CurResize, CurMove, CurLast }; /* cursor */
--enum { SchemeNorm, SchemeSel }; /* color schemes */
-+enum { SchemeNorm, SchemeSel, SchemeTitle }; /* color schemes */
- enum { NetSupported, NetWMName, NetWMState, NetWMCheck,
-        NetWMFullscreen, NetActiveWindow, NetWMWindowType,
-        NetWMWindowTypeDialog, NetClientList, NetLast }; /* EWMH atoms */
-@@ -730,7 +730,7 @@ drawbar(Monitor *m)
- 
- 	if ((w = m->ww - sw - x) > bh) {
- 		if (m->sel) {
--			drw_setscheme(drw, scheme[m == selmon ? SchemeSel : SchemeNorm]);
-+			drw_setscheme(drw, scheme[m == selmon ? SchemeTitle : SchemeNorm]);
- 			drw_text(drw, x, 0, w, bh, lrpad / 2, m->sel->name, 0);
- 			if (m->sel->isfloating)
- 				drw_rect(drw, x + boxs, boxs, boxw, boxw, m->sel->isfixed, 0);
--- 
-2.20.1
-
