diff --git a/config.h b/config.h
index 83ced98..125a421 100644
--- a/config.h
+++ b/config.h
@@ -20,16 +20,24 @@ static int topbar             = 1;        /* 0 means bottom bar */
  * yay -Sy ttf-joypixels
  */
 static char *fonts[]          = { "monospace:size=10", "JoyPixels:pixelsize=10:antialias=true:autohint=true"  };
-static char normbgcolor[]           = "#181e24";
-static char normbordercolor[]       = "#222D32";
-static char normfgcolor[]           = "#b7c5d3";
+
+static char normfgcolor[]           = "#F9FAF9";
+static char normbgcolor[]           = "#222B33";
+static char normbordercolor[]       = "#222B33";
+
 static char selfgcolor[]            = "#171D23";
-static char selbordercolor[]        = "#539AFC";
 static char selbgcolor[]            = "#539AFC";
+static char selbordercolor[]        = "#539AFC";
+
+static char titlefgcolor[]          = "#b7c5d3";
+static char titlebgcolor[]          = "#181E24";
+static char titlebordercolor[]      = "#181E24";
+
 static char *colors[][3] = {
-       /*               fg           bg           border   */
-       [SchemeNorm] = { normfgcolor, normbgcolor, normbordercolor },
-       [SchemeSel]  = { selfgcolor,  selbgcolor,  selbordercolor  },
+       /*                 fg            bg            border   */
+       [SchemeNorm]   = { normfgcolor,  normbgcolor,  normbordercolor   },
+       [SchemeSel]    = { selfgcolor,   selbgcolor,   selbordercolor    },
+       [SchemeTitle]  = { titlefgcolor, titlebgcolor, titlebordercolor  },
 };
 
 typedef struct {
diff --git a/dwm.c b/dwm.c
index 4d725ea..bddab5f 100644
--- a/dwm.c
+++ b/dwm.c
@@ -71,7 +71,7 @@
 
 /* enums */
 enum { CurNormal, CurResize, CurMove, CurLast }; /* cursor */
-enum { SchemeNorm, SchemeSel }; /* color schemes */
+enum { SchemeNorm, SchemeSel, SchemeTitle }; /* color schemes */
 enum { NetSupported, NetWMName, NetWMState, NetWMCheck,
        NetWMFullscreen, NetActiveWindow, NetWMWindowType,
        NetWMWindowTypeDialog, NetClientList, NetLast }; /* EWMH atoms */
@@ -912,7 +912,7 @@ drawbar(Monitor *m)
 
 	if ((w = m->ww - tw - x) > bh) {
 		if (m->sel) {
-			drw_setscheme(drw, scheme[m == selmon ? SchemeSel : SchemeNorm]);
+			drw_setscheme(drw, scheme[m == selmon ? SchemeTitle : SchemeNorm]);
 			drw_text(drw, x, 0, w, bh, lrpad / 2, m->sel->name, 0);
 			if (m->sel->isfloating)
 				drw_rect(drw, x + boxs, boxs, boxw, boxw, m->sel->isfixed, 0);
